src/createServer.js

```diff
@@ -1,3 +1,42 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const fs = require('fs');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const fileName = req.url.slice(1);
+    const urlParams = new URL(req.url, `http://${req.headers.host}`);
+    const pathName = urlParams.pathname;
+
+    if (pathName === '/favicon.ico') {
```

Comment:
If the pathName is '/favicon.ico', the function returns without sending any response. It's better to send a 404 status code and a message indicating that the favicon is not found.